common/models/coffee-shop.js

```diff
@@ -1,3 +1,6 @@
+// Copyright Owner 2014,2018. All Rights Reserved.
+// Node module: loopback-getting-started
+
```

Comment:
It's missing something like below:
```
// This file is licensed under the MIT License.
// License text available at https://opensource.org/licenses/MIT
```
And the "author" is not correct. Could you please check again? thanks.

Reply:
fixed owner name. This repo doesn't have MIT license info so it won't be included.

Reply:
> It's missing something like below:
> 
> ```
> // This file is licensed under the MIT License.
> // License text available at https://opensource.org/licenses/MIT
> ```
> 
> And the "author" is not correct. Could you please check again? thanks.

fixed owner name. This repo doesn't have MIT license info so it won't be included.